openpdf/src/main/java/com/lowagie/text/pdf/PdfNumber.java

```diff
@@ -62,7 +64,7 @@
  * @see        PdfObject
  * @see        BadPdfFormatException
  */
-public class PdfNumber extends PdfObject {
+public class PdfNumber extends PdfObject implements Comparable {
```

Comment:
Why raw `Comparable` if `PdfNumber` can only be compared with other `PdfNumber`s?

Reply:
In the com.lowagie.text.pdf.PdfPageLabels#getPageLabelFormats method, there is a Arrays.sort(numbers) call. If PdfNumber doesn't implement Comparable, then it won't work.

Reply:
I agree, will fix this.